Backend/src/app.ts

```diff
@@ -32,6 +33,15 @@ class App {
     this.express.use(express.urlencoded({ extended: true }));
     this.express.use(helmet());
     this.express.use(express.static('public'));
+    this.express.use(
+      cookieSession({
+        maxAge: 24 * 60 * 60 * 1000,
```

Comment:
More readable if they are ordered
seconds * minutes * hours * days
```suggestion
        maxAge:  60 * 60 * 24 * 1000,
```